wasm2c/README.md

```diff
@@ -327,6 +349,23 @@ arguments and returning `void` . e.g.
 
 `wasm_rt_free_memory` frees the memory instance.
 
+`wasm_rt_allocate_memory_shared` initializes a memory instance that can be
+shared by different Wasm instances, and allocates at least enough space for the
+given number of initial pages. The memory must be cleared to zero. The `is64`
+parameter indicates if the memory is indexed with an i32 or i64 address.
+
+`wasm_rt_grow_memory_shared` must grow the given shared memory instance by the
```

Comment:
Can this reference the (very similar) description of `wasm_rt_grow_memory` instead of duplicating the text?